docs/changelog/1727.feature.rst

```diff
@@ -0,0 +1 @@
+Support parsing [tox:testenv*] sections in setup.cfg. - by :user:`mauvilsa`
```

Comment:
Perhaps the right wording here is officially/explicitly. Because the support is already there, just not spelled out.

Reply:
Support with the `tox:` prefix is not there yet.